pandas/tests/scalar/test_period.py

```diff
@@ -13,7 +13,8 @@
 from pandas._libs import tslib, period as libperiod
 from pandas._libs.tslibs.parsing import DateParseError
 from pandas import Period, Timestamp, offsets
-from pandas.tseries.frequencies import DAYS, MONTHS
+from pandas._libs.tslibs.resolution import DAYS
+from pandas.tseries.frequencies import MONTHS
```

Comment:
why doesn't MONTHS simply exist in resolution?

Reply:
alias naming, will change